settings.py

```diff
@@ -2093,3 +2093,5 @@ def _pkce_required(client_id):
 
 # NOTE: if you are adding a new setting that you intend to have other environments override,
 # make sure you add it before localsettings are imported (from localsettings import *)
+
+MAX_GEOSPATIAL_INDEX_DOC_LIMIT = 1000000
```

Comment:
I the number should be fine, but we should atleast have some benchmarks on the number, if it affects the ES cluster in anyway.

Reply:
Sounds good. I'll see if QA is able to do some load testing, otherwise I'll look into doing this.

Reply:
No need to change this here, but just useful to know about. [Since Python 3.6](https://peps.python.org/pep-0515/), you can use underscores to separate thousands in decimals (and words in hexadecimals and nibbles in binaries) to make numbers more readable. e.g.

```python
MAX_GEOSPATIAL_INDEX_DOC_LIMIT = 1_000_000
addr = 0xCAFE_F00D
flags = 0b_0011_1111_0100_1110
```

Reply:
@kaapstorm Initially this is how I had it but then ultimately decided against it since I didn't see this formatting being used anywhere else within the `settings.py` file. I think this is fine to keep as is, but good to keep in mind for next time!